dashboard/src/components/AppView/AccessURLTable/AccessURLTable.tsx

```diff
@@ -90,6 +96,12 @@ class AccessURLTable extends React.Component<IAccessURLTableProps> {
     }
     return;
   }
+
+  private fetchIngresses() {
+    // Fetch all related Ingress resources. We don't need to fetch Services as
+    // they are expected to be watched by the ServiceTable.
+    this.props.ingressRefs.forEach(r => this.props.getResource(r));
```

Comment:
We don't need to worry about re-fetching ingresses we've previously fetched - I assume?

Reply:
not really, it will just update the info